model/task/task.go

```diff
@@ -2455,6 +2455,12 @@ func GetTimeSpent(tasks []Task) (time.Duration, time.Duration) {
 	return timeTaken, latestFinishTime.Sub(earliestStartTime)
 }
 
+var taskFailureStatuses []string = []string{
```

Comment:
I wonder if this list should in fact be https://github.com/evergreen-ci/evergreen/blob/bc1b972b2b963856a001e12c517ec68a42d2a61d/globals.go#L243-L250. It might make sense to make that list a variable in globals.go (and make `IsFailedTaskStatus` check that list). There may not have been task setup failures in the example data, for example, but they can occur.

Reply:
I originally included these statuses but then removed them on account the UI considers them "system issues". The failures in this list are based on what the UI considers a failure in the patch tasks table status dropdown menu. the failure statuses for which are documented in the code [here](https://github.com/tgrander/spruce/blob/master/src/pages/patch/patchTabs/tasks/TaskFilters.tsx#L123).

Now that you bring it up tho i agree that any finished, non-passing status should be sorted to the top. Will add in.